services/core-web/src/components/mine/Variances/MineVarianceTable.js

```diff
@@ -109,102 +107,105 @@ export class MineVarianceTable extends Component {
-      {
-        title: "Code Section",
-        dataIndex: "compliance_article_id",
-        sortField: "compliance_article_id",
         width: 150,
-        render: (text) => <div title="Code Section">{text}</div>,
-        sorter: !this.props.isDashboardView
-          ? (a, b) => compareCodes(a.compliance_article_id, b.compliance_article_id)
-          : true,
+        render: (text) => <div title="Number">{text}</div>,
       },
       {
-        title: "Mine Name",
-        dataIndex: "mineName",
+        title: "Mine",
+        dataIndex: "mine_name",
         sortField: "mine_name",
+        sorter: this.props.isDashboardView,
         width: 150,
         className: hideColumn(!this.props.isDashboardView),
         render: (text, record) => (
-          <div title="Mine Name" className={hideColumn(!this.props.isDashboardView)}>
-            <Link to={router.MINE_SUMMARY.dynamicRoute(record.mineGuid)}>{text}</Link>
+          <div title="Mine" className={hideColumn(!this.props.isDashboardView)}>
+            <Link to={router.MINE_SUMMARY.dynamicRoute(record.mine_guid)}>{text}</Link>
           </div>
         ),
+      },
+      {
+        title: "Code",
+        dataIndex: "compliance_article_id",
+        sortField: "compliance_article_id",
         sorter: this.props.isDashboardView,
+        width: 150,
+        render: (text) => (
+          <div title="Code">{this.props.complianceCodesHash[text] || Strings.EMPTY_FIELD}</div>
+        ),
       },
       {
         title: "Lead Inspector",
-        dataIndex: "leadInspector",
+        dataIndex: "lead_inspector",
         sortField: "lead_inspector",
+        sorter: this.props.isDashboardView,
         width: 150,
         className: hideColumn(!this.props.isDashboardView),
-        render: (text) => (
-          <div title="Mine Name" className={hideColumn(!this.props.isDashboardView)}>
-            {text}
-          </div>
-        ),
-        sorter: this.props.isDashboardView,
+        render: (text, record) =>
+          (record.inspector_party_guid && (
+            <Link
+              to={router.PARTY_PROFILE.dynamicRoute(record.inspector_party_guid)}
+              title="Lead Inspector"
+            >
+              {text}
+            </Link>
+          )) || <div title="Lead Inspector">{text}</div>,
       },
       {
         title: "Submission Date",
         dataIndex: "received_date",
         sortField: "received_date",
+        sorter: this.props.isDashboardView,
         width: 150,
         className: hideColumn(!this.props.isApplication),
-        render: (text) => (
-          <div className={hideColumn(!this.props.isApplication)} title="Submission Date">
-            {text}
-          </div>
-        ),
-        sorter: !this.props.isDashboardView
-          ? (a, b) => (moment(a.received_date) > moment(b.received_date) ? -1 : 1)
-          : true,
-        defaultSortOrder: "ascend",
+        render: (text) => <div title="Submission Date">{text}</div>,
```

Comment:
`className={hideColumn(!this.props.isApplication)}` must be on the div as well, otherwise this section will be visible on responsive tables